tests/integration.go

```diff
@@ -172,9 +160,6 @@ func buildkitVersion(t *testing.T, sb integration.Sandbox) string {
 			require.NoError(t, err, "undock not found")
 
 			destDir := t.TempDir()
-			t.Cleanup(func() {
-				os.RemoveAll(destDir)
-			})
```

Comment:
Not really related, but my IDE was complaining about the unhandled error, and it was redundant, so added a commit to remove it 😂